modules/web/src/app/node-data/component.ts

```diff
@@ -339,7 +339,7 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     return provider.includes(this.provider);
   }
 
-  // Source of truth for supported operating system: https://github.com/kubermatic/machine-controller/blob/main/docs/operating-system.md
+  // Source of truth for supported operating system: https://k8c.io/machine-controller/blob/main/docs/operating-system.md
```

Comment:
Please revert this change

Reply:
Ah yes, whoops. I saw it 3 times and thought "nice" every time.. 🙈